src/BuildConfig.cc

```diff
@@ -541,6 +603,10 @@ BuildConfig::setVariables() {
   for (const std::string_view flag : profile.cxxflags) {
     cxxflags.emplace_back(flag);
   }
+  for (const std::string& flag : getEnvFlags("CXXFLAGS")) {
+    cxxflags.emplace_back(flag);
+    logger::debug("adding env: ", flag);
```

Comment:
Do we need this?

Reply:
Cleaned up in the latest commit. (Just for testing when I'm playing around with the code and I forgot to remove it. I wrote a test for `parseEnvFlags` in the latest commit so now I can test it via `poac test`)